monai/networks/blocks/selfattention.py

```diff
@@ -52,8 +54,11 @@ def __init__(
             raise ValueError("hidden size should be divisible by num_heads.")
 
         self.num_heads = num_heads
-        self.out_proj = nn.Linear(hidden_size, hidden_size)
-        self.qkv = nn.Linear(hidden_size, hidden_size * 3, bias=qkv_bias)
+        self.dim_head = hidden_size // num_heads if dim_head is None else dim_head
```

Comment:
Will this one change `self.head_dim` in Line66?

Reply:
Thanks for the quick update, could you please also some test in this file: https://github.com/Project-MONAI/MONAI/blob/dev/tests/test_selfattention.py?

Thanks!

Reply:
Tests added in 55d0cdccada36564816cd1deecde43214457514e